ush/err_exit

```diff
@@ -1,13 +1,12 @@
-#!/bin/sh
+#!/bin/bash
 
-# SCRIPT NAME:  err_exit  Author:   Bill Facey   2 Apr 96
-#                         Modified: Kit Menlove 28 May 15
```

Comment:
I don't think you should take out the names of the original authors of the code.

Reply:
Sounds fair to leave the original authors, this was the change coming from https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/prod_util.v2.0.14/

Reply:
OK, we'll just leave it exactly as they had it, to make it easier to merge future changes.